### DISCUSSION TITLE
Deleted Tenant Event

### DISCUSSION THREAD

**Discussion by evgeniy-kryazhev** [abpframework/abp#12673] [General]
When deleting a tennant, is there an event that can be caught, something like Distributed Event Bus

**Comment by berkansasmaz** [comment]
Yes, there is :)

See: https://docs.abp.io/en/abp/latest/Distributed-Event-Bus#pre-defined-events

`IDistributedEventHandler<EntityDeletedEto<TenantEto>>`
